src/core/etl/tests/Flow/ETL/Tests/Benchmark/EntryFactory/NativeEntryFactoryBench.php

```diff
@@ -19,35 +18,30 @@ public function bench_entry_factory(array $params) : void
 
     public function provideRows() : \Generator
     {
-        $faker = Factory::create();
-
         $callback = static fn (int $i) : array => [
-            'order_id' => $faker->uuid,
-            'created_at' => $faker->dateTimeThisYear->format(\DateTimeInterface::RFC3339),
-            'updated_at' => $faker->dateTimeThisMonth->format(\DateTimeInterface::RFC3339),
-            'cancelled_at' => ($i % 10) === 0 ? $faker->dateTimeThisMonth->format(\DateTimeInterface::RFC3339) : null,
+            'order_id' => $i,
+            'created_at' => (new \DateTimeImmutable("-{$i} day ago"))->format(\DateTimeInterface::RFC3339),
+            'updated_at' => (new \DateTimeImmutable("+{$i} day"))->format(\DateTimeInterface::RFC3339),
+            'cancelled_at' => ($i % 10) === 0 ? (new \DateTimeImmutable("+{$i} day"))->format(\DateTimeInterface::RFC3339) : null,
             'active' => !(($i % 20) === 0),
-            'total_price' => $faker->randomFloat(2, 0, 500),
-            'discount' => $faker->randomFloat(2, 0, 50),
+            'total_price' => \random_int(2, 500),
+            'discount' => \random_int(0, 50),
             'customer' => [
-                'name' => $faker->firstName,
-                'last_name' => $faker->lastName,
-                'email' => $faker->email,
+                'name' => 'firstName_' . $i,
+                'last_name' => 'lastName_' . $i,
+                'email' => \base64_encode(\random_bytes(10)),
             ],
             'address' => [
-                'street' => $faker->streetAddress,
-                'city' => $faker->city,
-                'zip' => $faker->postcode,
-                'country' => $faker->country,
+                'street' => 'streetAddress_' . $i,
+                'city' => 'city_' . $i,
+                'zip' => 'zip_' . $i,
+                'country' => 'country_' . $i,
                 'location' => [
-                    'lat' => $faker->latitude,
-                    'lng' => $faker->longitude,
+                    'lat' => ($i + 1) / 2,
+                    'lng' => ($i + 1) / 4,
                 ],
             ],
-            'notes' => \array_map(
-                static fn ($i) => $faker->sentence,
-                \range(1, 3)
-            ),
+            'notes' => \base64_encode(\random_bytes(\random_int(10, 200))),
```

Comment:
I think we should fix the dataset to constant values, randomizing values in benchmarks might give us random results, I would probably just put here static data since values does not matter in this case.